_posts/2021-01-04-acidanthera-january.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title:  "Acidanthera Updates: January 2021"
+date:   2021-01-04 15:00:00 -0700
+categories: Hackintosh updates
+---
+
+* [A message from vit9696](#a-message-from-vit9696)
+* [Dortania Updates](#dortania-updates)
+* [Changelogs](#changelogs)
+
+# A message from vit9696
+
+Happy New Year everyone! We hope that you had and perhaps still have some calm holidays and are ready to check-in for a small but important update we prepared. We begin this year under the slogan of stability and usability improvements, and this time we did really well to help you maintain your setup.
+
+To begin with, [@PMheart](https://github.com/PMheart) wrote an absolutely overwhelming update to [ocvalidate utility](https://github.com/acidanthera/OpenCorePkg). While some of you might not know, ocvalidate is an OpenCore configuration checking tool, which only reports issues that are clear violations of the specification, i.e. real mistakes in the configuration. [@PMheart](https://github.com/PMheart) added many semantical checks that allow not only finding missing fields or wrong field types, but also various inconsistencies, incorrect values, duplicate entries, and many more. Hopefully, ocvalidate will continue to ensure a cleaner configuration for everyone. Other than that, several other utilities like [dmidecode](), [macserial, macrecovery, and LogoutHook](https://github.com/acidanthera/OpenCorePkg) got their own updates with many thanks going to [@roddy20](https://github.com/roddy20) for still caring about older hardware.
+
+It was quite some time we made changes to [OpenCanopy](https://github.com/acidanthera/OpenCorePkg), so this update brings small but important changes to its audio-visual experience. From now on you can utilize prefixes for the icon packs, and as a result, a new Modern icon pack was bundled as a built-in addendum thanks to [@Andrey1970AppleLife](https://github.com/Andrey1970AppleLife). We are continuing to make the artists' lives better, and hopefully, this change will make their experience even more smooth. In addition, we also brought audio assistant support to OpenCanopy and re-encoded the audio files significantly reducing their size.
```

Comment:
```suggestion
It has been quite some time since we made changes to [OpenCanopy](https://github.com/acidanthera/OpenCorePkg), so this update brings small but important changes to its audio-visual experience. From now on you can utilize prefixes for the icon packs, and as a result, a new Modern icon pack was bundled as a built-in addendum thanks to [@Andrey1970AppleLife](https://github.com/Andrey1970AppleLife). We are continuing to make the artists' lives better, and hopefully, this change will make their experience even more smooth. In addition, we also brought audio assistant support to OpenCanopy and re-encoded the audio files, significantly reducing their size.
```